src/main/java/com/microsoft/aad/msal4j/Account.java

```diff
@@ -28,6 +28,10 @@
 import lombok.Setter;
 import lombok.experimental.Accessors;
 
+/**
+ * Representation of a single user account. If modifying this object, ensure it is compliant with
+ * cache persistent model
```

Comment:
should this be in the public docs? That seems like an internal comment?

Reply:
Removed comment